### PR TITLE
Adding small Select prop and styling to basic single and multiple Select

### PR DESCRIPTION
**Pull request by mrazian85**
[Link to updated components in Deploy Preview](https://deploy-preview-97--neo-react-library-storybook.netlify.app/?path=/story/components-select--small-selects)

**Before tagging the team for a review, I have done the following:**
- [x] reviewed my code changes
- [x] ensured that all tests pass
- [x] updated the link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [x] tagged Matt if any visual changes have occurred
- [x] done an accessibility check on my work

Adding prop for new Small Select class to regular Single and Multiple Select Components as per [NEO-1249](https://jira.forge.avaya.com/browse/NEO-1249) related work done in latest version of Neo CSS library.

### CODE REVIEW

File: src/components/Select/Select.stories.tsx [hunk 1]
```diff
@@ -356,3 +356,25 @@ export const MoreThanOneMultipleSelect = () => {
     </Sheet>
   );
 };
+
+export const SmallSelects = () => {
```
Comment (yangAtSpoken) [inline src/components/Select/Select.stories.tsx:360]: Please add this to unit test.

Comment (mrazian85) [reply src/components/Select/Select.stories.tsx:360]: ✅

**Comment by mrazian85** [comment]
> Noticed that the dropdown doesn't move with the tab selection as it gets cut off. It seems to only work if I enable my screen reader.

Hey @matthewAvaya, this PR is just to add the option to toggle the Small Select class; it doesn't affect the way the Component handles tabbing or any other events and I didn't make any changes in that respect.

So, I guess I should open a separate ticket to do that work if there are improvements that need to be made 🙏

**Review by enrique-prado — APPROVED** [review]

**Comment by matthewAvaya** [comment]
I see. Yes, please add new tickets regarding the new issues. Thanks!

**Review by matthewAvaya — APPROVED** [review]

**Review by yangAtSpoken — APPROVED** [review]

**Pull request merged**